modules/sdk-lib-mpc/test/unit/shamir.ts

```diff
@@ -1,17 +1,19 @@
-import { Ed25519Curve, ShamirSecret, Secp256k1Curve } from '@bitgo/sdk-core';
+import 'should';
```

Comment:
```suggestion
```
Should you maybe replace any `should` use with `assert`?

Reply:
This PR doesn't actually change the tests/ src code in any meaningful way, just moving things around + some renaming,